vsts/vsts/vss_client.py

```diff
@@ -12,7 +12,7 @@
 
 from msrest import Deserializer, Serializer
 from msrest.exceptions import DeserializationError, SerializationError
-from msrest.pipeline import ClientRequest
+from msrest.universal_http.requests import ClientRequest
```

Comment:
looks like this should be just:
`from msrest.universal_http import ClientRequest` without the `.requests`

We will need to update the dependency versions also:
`"msrest>=0.6.0,<0.7.0"`